### DISCUSSION TITLE
CRS EPSG:4979 is not supported

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#984] [Configuration] [answered]
Hi,
I upgraded ldproxy from v 3.3.1 to **3.4** today and get errors of the type:
ERROR [2023-05-31 09:45:11,648]                          - PROJ: Could not initialize class org.kortforsyningen.proj.NativeResource
ERROR [2023-05-31 09:45:11,648]                          - Error starting CrsTransformerFactoryProj: Could not initialize class org.kortforsyningen.proj.NativeResource
**ERROR [...] Cannot load 'entities/services/ankringsforhold.yml': CRS EPSG:4979 is not supported.**
or
**ERROR [...] Cannot load 'entities/services/ankringsforhold.yml': Could not initialize class org.kortforsyningen.proj.ObjectFactory**

The feature provider seems to start in the log, but I get a HTTP ERROR 500 Internal Server Error

I cannot see that I have configured with the code 4979 at all.
Any idea where to look?

Kind Regards Sverre Iversen

**Comment by cportele** [comment]
I have these errors sometimes in my development environment when starting ldproxy, typically after changing the branch. This is triggered by a failure to load PROJ for some reason. We have not yet investigated the cause, because in my experience a) this problem always goes away on another start and b) we have not seen this happening in any of our deployments.

Questions:

* Does the problem persist on repeated starts?
* How did you start ldproxy and in which environment / on which OS?

**Reply by cportele** [reply, marked as answer]
I had a closer look. It should be a timing issue, PROJ is used during startup before it is ready. I have pushed a change in #986. This should be included in v3.4.1 scheduled for mid June. For now, you can test it with image `iide/ldproxy:next`.

**Reply by Sverre-Iversen** [reply]
Wow, that did the trick :)
Thanks
